<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OuvrageQtClass</class>
 <widget class="QDialog" name="OuvrageQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OuvrageQt</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_auteurOuvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_titreOuvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_villeOuvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_editeurOuvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_identifiantOuvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_anneeOuvrage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>2019</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du ou des auteurs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre de l'ouvrage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville d'édition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'éditeur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Année de publication</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter les informations de l'ouvrage à ajouter à la bibliographie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouterOuvrage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter l'ouvrage</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
